IDEApplication.cpp

```diff
@@ -138,8 +138,8 @@ void IDEApplication::initializePlugins()
 void IDEApplication::initializeTranslator()
 {
     if(! mTranslator.load("arduide_" + QLocale::system().name(), TRANSLATION_PATH))
-     {
-      qDebug() << "loading translator failed.";
-     }
+    {
+        qDebug() << "loading translator failed.";
+    }
```

Comment:
This change is unrelated, this is something one should generally avoid.

No problem for this time though.